Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -527,18 +527,20 @@ static bool sampleBufferRenderersSupportKeySession()
 
 void SourceBufferPrivateAVFObjC::didUpdateFormatDescriptionForTrackId(Ref<TrackInfo>&& formatDescription, uint64_t trackId)
 {
-    if (is<VideoInfo>(formatDescription)) {
-        auto result = m_videoTracks.find(AtomString::number(trackId));
-        if (result != m_videoTracks.end())
-            result->value->setFormatDescription(downcast<VideoInfo>(formatDescription.get()));
-        return;
-    }
```

Comment:
didUpdateFormatDescriptionForTrackId name is a bit weird.
I know this is consistent, but it would seem better to rename it as appendUpdateFormatDescriptionTask() or something like this.